bin/handler-sendgrid.rb

```diff
@@ -7,7 +7,7 @@
 # Released under the same terms as Sensu (the MIT license); see LICENSE
 # for details.
 
-require 'rubygems' if RUBY_VERSION < '1.9.0'
+require 'rubygems' if RUBY_VERSION < '2.0.0'
```

Comment:
You can actually drop this line as it will fail to install as in the gemspec we declare that we only support ruby `>= 2.0.0`